### DISCUSSION TITLE
Win7 support?

### DISCUSSION THREAD

**Discussion by mrMiiao** [lapce/lapce#1533] [Q&A] [answered]
Something without ResizePseudoConsole?

Does somebody know which dependency uses this function from kernel32?

**Comment by panekj** [comment, marked as answer]
Yes, Alacritty

**Reply by mrMiiao** [reply]
Wow... thank you! Is there a way to get rid of it in a custom build?

I feel like a piece of bread...

**Reply by panekj** [reply]
https://github.com/lapce/lapce/compare/master...panekj:lapce:feat/terminal

**Reply by mrMiiao** [reply]
Seems like it's not possible yet. Anyway, thank you!

**Comment by coffeeispower** [comment]
I'm gonna be honest with you, if you still use windows 7, I recommend you to switch to linux or at least upgrade your windows.
